integration-tools/src/main/java/uk/ac/bbsrc/tgac/miso/integration/dp5mirage/DP5MirageScanner.java

```diff
@@ -65,64 +68,65 @@ public BoxScan getScan() throws IntegrationException{
         }
         else if (response.statusCode() == 459) {
-          throw new IntegrationException(String.format("%s The scan is not found. Try rescanning "
-              + "after a short period of time.", errorMessage));
+          throwScanError("The scan is not found. Try rescanning after a short period of time",
+              true);
         }
         else if (response.statusCode() == 461) {
-          throw new IntegrationException(String.format("%s Scanner not connected. %s",
-              errorMessage, actionMessage));
+          throwScanError("The Scanner is not connected. Please check the connection before "
+              + "rescanning",true);
         }
         else if (response.statusCode() == 468) {
-          throw new IntegrationException(String.format("%s Scan result is not ready. Try "
-                  + "rescanning after a short period of time.",
-              errorMessage));
+          throwScanError("Scan result is not ready" ,false);
         }
         else if (response.statusCode() == 477) {
-          throw new IntegrationException(String.format("%s Failed to read a barcode. %s",
-              errorMessage, actionMessage));
+          throwScanError("Failed to read a barcode" ,false);
         }
         else if (response.statusCode() == 478) {
-          throw new IntegrationException(String.format("%s Scanner not found. %s", errorMessage,
-              actionMessage));
+          throwScanError("Scanner not found" ,false);
         }
         else if (response.statusCode() == 488) {
-          throw new IntegrationException(String.format("%s Linear Reader is not configured. %s",
-              errorMessage, actionMessage));
+          throwScanError("Linear Reader is not configured" ,false);
         }
         else {
-          throw new IntegrationException(errorMessage);
+          throwScanError("", false);
         }
       }
 
       // Parse JSON into a JsonNode
       ObjectMapper mapper = new ObjectMapper();
       JsonNode rootNode = mapper.readTree(response.body());
-      JsonNode barcodesNode = rootNode.get("tubeBarcode");
 
       // Convert the 'barcodes' array into a list of barcodePositionData records
-      records = mapper.convertValue(barcodesNode, new TypeReference<>() {});
+      records = mapper.convertValue(rootNode.get("tubeBarcode"), new TypeReference<>() {});
 
     } catch (IOException | InterruptedException | URISyntaxException e) {
       throw new IntegrationException("Error communicating with the scanner", e);
     }
 
     return records == null ? null : new DP5MirageScan(records);
   }
+
+  // Used to help determine what the problem is for an unexpected scan response
+  private static void throwScanError(String specificMessage, boolean reportSpecific)
+      throws IntegrationException {
+    if (reportSpecific) {
+      throw new IntegrationException("Error Reported by the DP5 scanner. " + specificMessage);
+    } else {
+      log.info("Timestamp: {}", new Date());
```

Comment:
Remove. The log should already have timestamps and a timestamp that's not attached to anything isn't useful